tests/test_annotation_stores.py

```diff
@@ -2091,6 +2116,7 @@ def test_nquery_overlapping_grid_poly_poly(store_cls):
 
     @staticmethod
     def test_invalid_mode_type(store_cls):
+        """Tests invalid mode type for AnnotationStore."""
```

Comment:
```suggestion
        """Test invalid mode type for AnnotationStore."""
```

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/625/commits/d5bc38668bd092f49acc621c6086ff35776bb23f